### DISCUSSION TITLE
Create .intunewin packages without connection to Tenant?

### DISCUSSION THREAD

**Discussion by glca18** [rink-turksma/IntunePrepTool#6] [Q&A] [answered]
Can I use this tool locally to create .intunewin packages, install/uninstall commands, or scripts without connecting it to my tenant?

I want to manually upload it to the tenants instead of this tool doing it for me.

**Comment by rink-turksma** [comment, marked as answer]
> Can I use this tool locally to create .intunewin packages, install/uninstall commands, or scripts without connecting it to my tenant?
> 
> I want to manually upload it to the tenants instead of this tool doing it for me.

Yes, you can. Just create a package. You don’t need to upload it directly.
When you create a package, the source files (and scripts) are stored in the package folder.
After clicking Create Intunewin, the Intunewin file will be generated for you in the output folder.
The config.csv file will also be there, where you can find details such as the detection method, etc.
